tljh/jupyterhub_config.py

```diff
@@ -34,7 +34,7 @@ def start(self):
         else:
             user.remove_user_group(system_username, 'jupyterhub-admins')
         return super().start()
-
+c.NotebookApp.notebook_dir = '/home/efs/'
```

Comment:
I don't think this is what we want for the default behavior. Did you mean to update your own configuration?